api/envoy/service/auth/v2/attribute_context.proto

```diff
@@ -108,7 +109,7 @@ message AttributeContext {
     // the URL path and query-string. No decoding is performed.
     string path = 4;
 
-    // The HTTP request `Host` or 'Authority` header value.
+    // The HTTP request `Host` or `Authority` header value.
```

Comment:
in this case i think it definitely should be a literal - so could you use double backticks please